src/StringCompression.h

```diff
@@ -32,8 +33,9 @@ namespace Compression
 		/** Returns a view (of type std::byte) of the internal store. */
 		ContiguousByteBufferView GetView() const;
 
-		/** A store allocated on either the stack or heap. */
-		std::variant<Static, Dynamic> Storage;
+		/** A store allocated on the heap. */
+		/** Note that this remains a variant for now. */
+		std::variant<Dynamic> Storage;
```

Comment:
Yes, this previously needed the `std::variant` because of needing to accommodate either a `std::array<std::byte>` or `std::unique_ptr<std::byte[]>` for type safety.

To change `Storage` to a std::vector<std::byte> would need some refactoring in `./src/StringCompression.cpp`, but it would be good to simplify and to boost performance.

Would this be raised as an additional issue on git after this current issue?